casual/casual-inbound-handler-casual-service/build.gradle

```diff
@@ -19,9 +19,11 @@ dependencies {
   implementation project(":casual:casual-inbound-handler-api")
   implementation project(":casual:casual-service-discovery-extension")
   compileOnly libs.javaee_api
-
+  compileOnly libs.gson
+    
   testImplementation libs.javaee_api
   testImplementation libs.gson
+  testImplementation libs.system_lambda
```

Comment:
Is system lambda actually ever used? I can't see it in any tests in this version, and seems to work fine locally even if I remove it

Reply:
It was earlier and now it is again :)